src/dash-table/components/Table/props.ts

```diff
@@ -316,6 +316,8 @@ interface IDefaultProps {
     css: IStylesheetRule[];
     data: Data;
     editable: boolean;
+    export_format: string;
+    export_header: string;
```

Comment:
`export_headers` instead

Reply:
In both cases you can be more precise and force the strings to be either enums or exact string values

```
export_format: 'csv' | 'xlsx'
export_headers: 'ids' | 'names' | 'none' | 'display'
```

or 

```
enum ExportFormat {
    CSV = 'csv',
    XLSX = 'xlxs'
}
export_format: ExportFormat
```